### DISCUSSION TITLE
Release schedule

### DISCUSSION THREAD

**Discussion by mrantivirus** [jonataslaw/getx#1510] [General]
I'm just curious is there is any sort of release schedule for this package? For a project I'm working on, we started switching to Get from Dio, but now hit a roadblock because `withCredentials` isn't configurable.

I saw that the changes were merged, but since there hasn't been a new release in almost 2 months, I was wondering if something is delaying it or if there is some sort of release schedule?

**Comment by nickwri** [comment]
I am also interested in learning when the next release is planned - hoping it will resolve #1426

**Comment by jwelmac** [comment]
How do you handle testing GetConnect?
